api/package.json

```diff
@@ -15,12 +15,14 @@
     "toc": "doctoc --github --maxlevel 2 README.md"
   },
   "dependencies": {
+    "@shared-libs/server-checks": "file:../shared-libs/server-checks",
     "@shared-libs/tombstone-utils": "file:../shared-libs/tombstone-utils",
     "@shared-libs/view-map-utils": "file:../shared-libs/view-map-utils",
     "async": "^2.6.0",
     "body-parser": "^1.18.2",
     "buffer-shims": "^1.0.0",
     "bulk-docs-utils": "file:../shared-libs/bulk-docs-utils",
+    "checks": "file:../shared-libs/server-checks",
```

Comment:
It looks like `../shared-libs/server-checks` is listed twice - that's a mistake, right?